src/Passes/Source/Rules/Factory.cpp

```diff
@@ -375,6 +419,11 @@ namespace quantum
         addRule({call("__quantum__qis__m__body", "qubit"_cap = _), std::move(replace_measurement)});
     }
 
+    void RuleFactory::optimiseBranchQuatumZero()
```

Comment:
Could you please do a search for "Quatum" everywhere in the folder and replace it with "Quantum"? Also, I would recommend using a spell check extension (e.g. https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker).